unlock-app/src/components/content/event/Form.tsx

```diff
@@ -0,0 +1,357 @@
+              />
+
+              <Input
+                {...register('metadata.image', {})}
+                type="url"
+                placeholder="Please enter an image URL"
+                label="Illustration"
+                description={
+                  'This illustration will be used for the NFT tickets.'
+                }
+              />
+
+              <Select
+                onChange={(newValue) => {
+                  setValue('network', Number(newValue))
+                  setValue('lock.currencyContractAddress', null)
+                  setValue(
+                    'currencySymbol',
+                    networks[newValue].baseCurrencySymbol
+                  )
+                }}
+                options={networkOptions}
+                label="Network"
+                defaultValue={network}
+                description={<NetworkDescription />}
+              />
+              <div className="mb-4">
+                {noBalance && (
+                  <BalanceWarning
+                    network={details.network!}
+                    balance={balance}
+                  />
+                )}
+              </div>
+            </div>
+          </Disclosure>
+
+          <Disclosure label="Location, date and time" defaultOpen>
+            <div className="grid gap-6">
+              <p className="mb-5">
+                This information will be public and included on each of the NFT
+                tickets. There again, it can be adjusted later.
+              </p>
+              <div className="grid items-center align-top	gap-4 sm:grid-cols-2">
+                <div className="flex flex-col self-start justify-top gap-4">
+                  <div className="h-80">
+                    <iframe width="100%" height="300" src={mapAddress}></iframe>
+                  </div>
+                </div>
+                <div className="flex flex-col self-start justify-top">
+                  <Input
+                    {...register('metadata.ticket.event_start_date')}
+                    type="date"
+                    label="Date"
+                  />
+                  <Input
+                    {...register('metadata.ticket.event_start_time')}
+                    type="time"
+                    label="Time"
+                  />
+
+                  <Controller
+                    name="metadata.ticket.event_timezone"
+                    control={control}
+                    render={({ field: { onChange, value } }) => {
+                      return (
+                        <Select
+                          onChange={(newValue) => {
+                            onChange({
+                              target: {
+                                value: newValue,
+                              },
+                            })
+                          }}
+                          // @ts-expect-error supportedValuesOf
+                          options={Intl.supportedValuesOf('timeZone').map(
```

Comment:
interesting. This seem to be a new addition because previously, you couldn't map over the Intl supported timezones.

Reply:
Yup!